string body_name
geometry_msgs/Point pose
---
bool success